client/src/components/Common/Layout/Stake/Table.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Table } from 'react-bootstrap';
+
+import { toFormattedNumber, displayNaN } from '../../../../helpers/format';
+
+import './Table.scss';
+
+const EmptyDelegation = () => (
+	<div className="cd_empty_delegation_row row">
+		<div className="cd_empty_delegation_section position-relative">
+			<img src="assets/image/no-staking-icon.svg" alt="empty-cspr-stake" />
+			<div className="cd_empty_delegation_message">
+				You do not have any delegations yet. Stake CSPR, earn rewards and help Capser become more secure!
+			</div>
+		</div>
+	</div>
+);
+
+const StakingAccountListComponent = ({ stakingDeployList = [] }) => {
+	if (!stakingDeployList.length) {
+		return <EmptyDelegation />;
+	}
+	const total = stakingDeployList
+		.filter((stake) => !!stake.successAmount)
+		.map((stake) => stake.successAmount)
+		.reduce((prev, next) => prev + next);
```

Comment:
Please add default value for reduce function. 
It throw error if array is empty
TypeError: Reduce of empty array with no initial value

Reply:
Updated.